### ISSUE TITLE
Test failure with PG12: Infinity/NaN

### ISSUE THREAD

**Issue by df7cb** [dvarrazzo/pgmp#14]
With #8 and #11 applied, the regression tests fail with PG12beta1:
```
15:23:27 **** regression.diffs ****
15:23:27 diff -U3 /tmp/autopkgtest.6ssk1w/tree/test/expected/mpz.out /tmp/autopkgtest.6ssk1w/tree/results/mpz.out
15:23:27 --- /tmp/autopkgtest.6ssk1w/tree/test/expected/mpz.out	2019-05-31 13:22:51.000000000 +0000
15:23:27 +++ /tmp/autopkgtest.6ssk1w/tree/results/mpz.out	2019-05-31 13:23:27.139763010 +0000
15:23:27 @@ -123,11 +123,11 @@
15:23:27  SELECT (-1e36::float4)::mpz BETWEEN -pow(10::mpz,36) - pow(10::mpz,30) AND -pow(10::mpz,36) + pow(10::mpz,30);
15:23:27  t
15:23:27  SELECT 'NaN'::float4::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "nan"
15:23:27 +ERROR:  can't convert float value to mpz: "NaN"
15:23:27  SELECT 'Infinity'::float4::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "inf"
15:23:27 +ERROR:  can't convert float value to mpz: "Infinity"
15:23:27  SELECT '-Infinity'::float4::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "-inf"
15:23:27 +ERROR:  can't convert float value to mpz: "-Infinity"
15:23:27  SELECT 0.0::float8::mpz, 123.15::float8::mpz, 123.95::float8::mpz;
15:23:27  0|123|123
15:23:27  SELECT (1e307::float8)::mpz BETWEEN pow(10::mpz,307) - pow(10::mpz,307-15) AND pow(10::mpz,307) + pow(10::mpz,307-15);
15:23:27 @@ -135,11 +135,11 @@
15:23:27  SELECT (-1e307::float8)::mpz BETWEEN -pow(10::mpz,307) - pow(10::mpz,307-15) AND -pow(10::mpz,307) + pow(10::mpz,307-15);
15:23:27  t
15:23:27  SELECT 'NaN'::float8::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "nan"
15:23:27 +ERROR:  can't convert float value to mpz: "NaN"
15:23:27  SELECT 'Infinity'::float8::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "inf"
15:23:27 +ERROR:  can't convert float value to mpz: "Infinity"
15:23:27  SELECT '-Infinity'::float8::mpz;
15:23:27 -ERROR:  can't convert float value to mpz: "-inf"
15:23:27 +ERROR:  can't convert float value to mpz: "-Infinity"
15:23:27  SELECT 0::mpz, 1::mpz, (-1)::mpz;       -- automatic casts
15:23:27  0|1|-1
15:23:27  SELECT 1000000::mpz, (-1000000)::mpz;
```

**Comment by dvarrazzo** [comment]
There can be an alternate regression result file to cater that with name suffix `_1`, e.g. see <https://github.com/reorg/pg_repack/tree/master/regress/expected>. Could you provide one in a MR please? I swear I'll apply it in a non distant future...

**Comment by df7cb** [comment]
Will do! (But not now...)

**Comment by df7cb** [comment]
Submitted as #15.

**Comment by dvarrazzo** [comment]
Fixed with #15
